app/assets/javascripts/visual-editor.js

```diff
@@ -1,2 +1,38 @@
 //= require govspeak-visual-editor/dist/govspeak-visual-editor.js
 
+(function () {
+  var content = document.querySelector(".js-visual-editor__content"),
+    container = document.querySelector(".js-visual-editor__container"),
+    textarea = document.querySelector(".js-visual-editor__textarea"),
+    markdownButton = document.querySelector(
+      ".js-visual-editor__markdown_button",
+    ),
+    markdownSection = document.querySelector(
+      ".js-visual-editor__markdown_section",
+    ),
+    copyButton = document.querySelector(".js-visual-editor__copy_button");
+
+  if (
+    !content ||
+    !container ||
+    !textarea ||
+    !markdownButton ||
+    !markdownSection ||
+    !copyButton
+  )
+    return;
+
+  new window.GovspeakVisualEditor(content, container, textarea, { images: [] });
+
+  markdownButton.addEventListener("click", () => {
+    event.preventDefault();
+    markdownSection.removeAttribute("hidden");
+    markdownSection.scrollIntoView({ behavior: "smooth" });
+  });
+
+  copyButton.addEventListener("click", () => {
+    event.preventDefault();
+    textarea.select();
+    document.execCommand("copy");
+  });
```

Comment:
I've had a go at both test approaches, but because the module is written in ES6, the tests don't work out of the box. I've put some WIP code in #533 and will pop it on our backlog to add something later (not business critical if the feature fails in the meantime).